figures/S1.supplemental_table1/results/2023_Interstellar_Table_S1.csv

```diff
@@ -0,0 +1,188 @@
+cytokine,apoptosis_healthy_p-adj,apoptosis_pyroptosis_p-adj,healthy_pyroptosis_p-adj,none_significant,apoptotic,pyroptotic,healthy,all_significant,putative_function
+AITRL (GITR Ligand),0.8458,0.5658,0.5388,none_significant, , ,, ,Not Annotated
```

Comment:
I noticed a pattern of " " spaces vs "" empty values. Would this be important to note or avoid somewhere? Feel free to ignore if this is unhelpful.